checkpoints/checkpoint2/checkpoint2/Portfolio/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+  <head>
+     <meta charset="UTF-8">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="./css/reset.css">
```

Comment:
2 issues.
1. reset css should come before your regular style.css other wise it would clear it.
2. your site continued to work, because your reset.css file was missing